src/components/AddToCart/AddToCart.js

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components';
+
+export default function AddToCart({currentmenu}) {
+  return (
+    <AddToCartContainer>
+      <Span>{currentmenu.price}€</Span>
```

Comment:
I would recommend to use camel case: 
```suggestion
export default function AddToCart({currentMenu}) {
  return (
    <AddToCartContainer>
      <Span>{currentMenu.price}€</Span>
```